app/views/idv/phone_errors/_warning.html.erb

```diff
@@ -19,6 +19,16 @@ locals:
           text: t('idv.troubleshooting.options.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
+        {
+          url: help_center_redirect_url(
+            category: 'verify-your-identity',
+            article: 'phone-number',
+            flow: :idv,
+            step: :phone,
```

Comment:
Below, there's also logic for a `location` property which can be provided as a logging detail.

```suggestion
            step: :phone,
            location: local_assigns.fetch(:name, 'warning'),            
```

Reply:
thank you! Added in 2 places.